ocean_provider/validation/algo.py

```diff
@@ -343,24 +343,18 @@ def validate(self):
             if not self.validate_algo():
                 return False
 
-        if asset_urls:
```

Comment:
can you please check whether asset_urls is still used anywhere? If it is not, please remove the resolve.